src/main/java/org/launchcode/techjobs/persistent/models/Id.java

```diff
@@ -0,0 +1,4 @@
+package org.launchcode.techjobs.persistent.models;
+
+public @interface Id {
+}
```

Comment:
```suggestion
```
I think this Id interface might have been created by accident while you added the annotation to one of your classes. Deleting this and importing `@Id` from `import javax.persistence*` will resolve the IllegalArgumentsException you're getting.